src/Immediate.Validations.Shared/Validators/EmptyAttribute.cs

```diff
@@ -22,7 +22,7 @@ public sealed class EmptyAttribute : ValidatorAttribute
 	/// <summary>
 	///		The default message template when the property is invalid.
 	/// </summary>
-	public const string DefaultMessage = "'{PropertyName}' must be empty.";
+	public static string DefaultMessage { get; } = ValidationConfiguration.Localizer[nameof(EmptyAttribute)].Value;
```

Comment:
This initializes a `static` value once, based on the culture at the time that the value is accessed for the first time. It should be a method-based property:
```suggestion
	public static string DefaultMessage => ValidationConfiguration.Localizer[nameof(EmptyAttribute)].Value;
```

Reply:
Fixed.